server/package.json

```diff
@@ -24,6 +24,8 @@
     "multer-s3": "^2.9.0",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
+    "react-dropzone": "^11.0.1",
```

Comment:
Shouldn't this be on the client instead?

Reply:
I might have installed it on the server end too, I'll fix it.